netbeans.apache.org/src/content/download/nb16/index.adoc

```diff
@@ -52,7 +52,7 @@ https://archive.apache.org/dist/netbeans/  (//archived)
 https://downloads.apache.org/netbeans/ (//current)
 ////
 
-Apache NetBeans {netbeans-version} was released on November 30, 2022. See link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Release Notes on GitHub] for all changes in Apache NetBeans {netbeans-version}.
+Apache NetBeans {netbeans-version} was released on November 30, 2022. See link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Release Notes on GitHub] for all changes.
```

Comment:
I would put the Changes in a separate section on this page even if that's just a link to GitHub

Reply:
done